jangaroo-browser/src/main/joo/js/FormData.as

```diff
@@ -0,0 +1,86 @@
+package js {
+
+/**
+ * The FormData interface provides a way to easily construct a set of key/value pairs representing form fields and their
+ * values, which can then be easily sent using the XMLHttpRequest.send() method. It uses the same format a form would
+ * use if the encoding type were set to "multipart/form-data".
+ *
+ * You can also pass it directly to the URLSearchParams constructor if you want to generate query parameters in the way
+ * a <form> would do if it were using simple GET submission.
+ *
+ * An object implementing FormData can directly be used in a for...of structure, instead of entries(): for (var p of
+ * myFormData) is equivalent to for (var p of myFormData.entries()).
+ */
+[Native("FormData")]
+public class FormData {
+
+  /**
+   * Creates a new FormData object.
+   * @param form
+   */
+  public native function FormData(form: HTMLFormElement = undefined);
+
+  /**
+   * Appends a new value onto an existing key inside a FormData object, or adds the key if it does not already exist.
+   *
+   * @param name The name of the field whose data is contained in value.
+   * @param value The field's value. This can be a {@link String} or {@link Blob} (including subclasses such as
+   *              {@link File}). If none of these are specified the value is converted to a string.
+   * @param fileName The filename reported to the server (a {@link String}), when a {@link Blob} or {@link File} is
+   *                 passed as the second parameter. The default filename for {@link Blob} objects is "blob". The
+   *                 default filename for {@link File} objects is the file's filename.
+   */
+  public native function append(name: String, value: *, fileName: String = undefined): void;
+
+  /**
+   * Deletes a key/value pair from a FormData object.
+   *
```

Comment:
```suggestion
   * Aliased by Jangaroo with an underscore suffix to avoid clash with ActionScript keyword
   * <code>delete</code>. To make this alias work, it is added to the class prototype.
   *
```

Same for all other ...`_` aliases, please.